app/presenters/edition_presenter.rb

```diff
@@ -83,10 +83,10 @@ def access_limited
     end
 
     def expanded_link_set_presenter
-      @expanded_link_set_presenter ||= Presenters::Queries::ExpandedLinkSet.new(
-        content_id: edition.content_id,
-        draft: draft,
-        locale: edition.locale
+      @expanded_link_set_presenter ||= Presenters::Queries::ExpandedLinkSet.by_content_id(
+        edition.content_id,
+        locale: edition.locale,
+        with_drafts: draft,
```

Comment:
hmm can we do this by_edition somehow? (sorry I'm sure this interface is the bain of your life) I'm guessing this was wrote before with_drafts was an option?